fix(docs): migration guide state saved to localstorage (#7505)
swisspost/design-system: .changeset/vast-pears-type.md

---
'@swisspost/design-system-documentation': patch
---

> Fixed a bug in the migration guide where each migration item's checkbox state was not saved correctly to localStorage.

Reviewer: ```suggestion
Fixed a bug in the migration guide where individual task checkboxes were not correctly saving their state to localStorage.
```
